helios-test/src/main/kotlin/helios/arrow/UnitSpec.kt

```diff
@@ -0,0 +1,26 @@
+package helios.arrow
+
+import helios.arrow.laws.Law
+import io.kotest.TestCase
+import io.kotest.TestType
+import io.kotest.specs.StringSpec
+
+
+/**
+ * Base class for unit tests
+ */
+abstract class UnitSpec : StringSpec() {
+
+    private val lawTestCases = mutableListOf<TestCase>()
+
+    fun testLaws(vararg laws: List<Law>): List<TestCase> = laws
+        .flatMap { list: List<Law> -> list.asIterable() }
```

Comment:
Is this line from Arrow? This is `flatMap(::identity)` which just returns itself since we're dealing with `List<Law>` and no nested containers.

Reply:
yes, it is from Arrow: https://github.com/arrow-kt/arrow/blob/master/modules/core/arrow-test/src/main/kotlin/arrow/test/UnitSpec.kt#L16